recipes/plink/meta.yaml

```diff
@@ -1,20 +1,20 @@
 {% set name = "plink" %}
-{% set version = "1.90b6.21" %}
+{% set version = "1.9.0b.7.7" %}
```

Comment:
This didn't work well.
https://anaconda.org/bioconda/plink still says that 1.90 is the latest version.
We have the same problem for another recipe too - https://github.com/bioconda/bioconda-recipes/pull/53947

Reply:
Hi, I just noticed this. Plink1.9 is going to be replaced by Plink2 eventually, so all these are more of bug fixes than active feature development so the version numbers are 1.9-x.x.x, but a lot of us use 1.9 over 2 at this time. 
Any thoughts on what should be done? I am very new to this. Thanks!

Reply:
https://anaconda.org/bioconda/plink/files
All previous versions were `1.90`. The one uploaded by this version is `1.9`. The resolver resolves `1.90` as the latest...
Since no one complained about all previous `1.90`s that they are no really correct I'd suggest to keep using `1.90` (or 1.91, or whatever makes sense).
If you agree please create a new PR that changes the version and resets the `build > number` to `0`.

Reply:
Thanks, can I instead make it "1.90b7.7"? I hope this will be higher than the "1.90b6.21".

Reply:
Yes, I think so